samples/uploadWithoutAuth.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function main(
+  bucketName = 'my-bucket',
+  contents = 'these are my file contents',
+  destFileName = 'file.txt'
+) {
+  // [START storage_upload_without_authentication]
```

Comment:
@cbonnie what do you think about this region tag name?

Reply:
Looks fine to me. I don't think the region tag is visible to the user.